locales/es-ES.json

```diff
@@ -39,28 +39,28 @@
 	"Editor Font Family": "Fuente del editor",
 	"Editor Indent Style": "Estilo de indentado del editor",
 	"Spaces": "Espacios",
-	"Tabs": "Tabulación",
-	"Switch to Preview": "Cambiar a Previsualización",
+	"Tabs": "Tabulaciónes",
+	"Switch to Preview": "Cambiar a previsualización",
 	"When Editor Blurred": "Cuando el editor pierde el foco",
 	"When Editor Blurred, Edit On Double Click": "Cuando el editor pierde el foco, editar con doble clic",
 	"On Right Click": "Al hacer clic derecho",
 	"Editor Keymap": "Mapeo de teclas del editor",
 	"default": "por defecto",
 	"vim": "vim",
 	"emacs": "emacs",
-	"⚠️ Please restart boostnote after you change the keymap": "⚠️ Reinicie boostnote después de cambiar el mapeo de teclas",
+	"⚠️ Please restart boostnote after you change the keymap": "⚠️ Por favor reinicie boostnote después de cambiar el mapeo de teclas",
 	"Show line numbers in the editor": "Mostrar números de línea en el editor",
 	"Allow editor to scroll past the last line": "Permitir al editor desplazarse más allá de la última línea",
-	"Bring in web page title when pasting URL on editor": "Al pegar una URL en el editor, insertar el título de la web automáticamente",
-	"Preview": "Previsualización",
+	"Bring in web page title when pasting URL on editor": "Al pegar una URL en el editor, insertar el título de la web",
+	"Preview": "Previsualizar",
 	"Preview Font Size": "Previsualizar tamaño de la fuente",
 	"Preview Font Family": "Previsualizar fuente",
 	"Code block Theme": "Tema de los bloques de código",
 	"Allow preview to scroll past the last line": "Permitir a la previsualización desplazarse más allá de la última línea",
-	"Show line numbers for preview code blocks": "Mostar números de línea al previsualizar bloques de código",
+	"Show line numbers for preview code blocks": "Mostrar números de línea al previsualizar bloques de código",
 	"LaTeX Inline Open Delimiter": "Delimitador de apertura LaTeX en línea",
 	"LaTeX Inline Close Delimiter": "Delimitador de cierre LaTeX en línea",
-	"LaTeX Block Open Delimiter": "Delimitado de apertura bloque LaTeX",
+	"LaTeX Block Open Delimiter": "Delimitador de apertura bloque LaTeX",
```

Comment:
"Delimitador de apertura de bloque LaTeX"

Reply:
Done, and I change the others too.
- "Delimitador de apertura de LaTeX en línea"
- "Delimitador de cierre de LaTeX en línea"
- "Delimitador de apertura de bloque LaTeX"
- "Delimitador de cierre de bloque LaTeX"